packages/server-core/src/media/storageprovider/s3ipfs.storage.ts

```diff
@@ -0,0 +1,306 @@
+import { FileContentType } from '@xrengine/common/src/interfaces/FileContentType'
+import AWS from 'aws-sdk'
+import { PresignedPost } from 'aws-sdk/clients/s3'
+import path from 'path'
+import S3BlobStore from 's3-blob-store'
+import config from '../../appconfig'
+import {
+  SignedURLResponse,
+  StorageListObjectInterface,
+  StorageObjectInterface,
+  StorageProviderInterface
+} from './storageprovider.interface'
+
+export class S3IPFSProvider implements StorageProviderInterface {
```

Comment:
what exact is different here from the s3 storage provider? seems like a lot of unnecessarily duplicated code. can we just use the s3 storage provider instead?